deid/tests/test_replace_identifiers_write.py

```diff
@@ -0,0 +1,892 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
Two quick questions here:

 - Do you know why this is rendering as a completely new file instead of rename of https://github.com/pydicom/deid/blob/master/deid/tests/test_replace_identifiers.py?
 - Why the rename?

Reply:
I went back and forth (and honestly, today, I'm on the other side).  Since the issue only appeared when the save=True in replace_identifiers, I thought that it could be beneficial to have a set of tests where save=True (test_replace_identifiers_write) and a set where save=False (test_replace_identifiers).  But really, this is redundant since shifting everything to write is still testing the "save=False" scenario as well.

Reply:
Can you point me to the case that had a different result for save vs not? I would suggest just testing the save_identifiers cases with each of save is True and save is False.

Reply:
By flipping the default Tag VR to "ST", save=True and save=False do behave the same (#146).  However when the default Tag was "UN", we saw different behavior in [test_add_private_constant](https://github.com/pydicom/deid/blob/bb98013b4dad4674af0e4f338562a6773d4f8132/deid/tests/test_replace_identifiers.py#L58) and [test_add_tag_variable](https://github.com/pydicom/deid/blob/bb98013b4dad4674af0e4f338562a6773d4f8132/deid/tests/test_replace_identifiers.py#L196).

I'm good with however you think is best to test this. My initial thought yesterday was to have a save=False version of the tests (test_replace_identifiers.py) and a save=True version of the tests (test_replace_identifiers_write.py) - that's what I initially submitted with the PR - but it is redundant. If you'd prefer it, I could just simply create second versions of test_add_private_constant and test_add_tag_variable.

Reply:
Yes I like that approach - second versions of specific tests that have issue, but for the rest we probably don’t need to save to file. Does that work?

Reply:
That's fine with me!  I'll commit it in a few.